docs/Troubleshooting.md

```diff
@@ -138,3 +138,99 @@ and is a JSON object generated by the Quilt build system.
 
     `$ aws cloudformation describe-stack-events --stack-name QUILT_STACK_NAME`
 
+## Collecting logs
+
+To expedite the resolution of any errors encountered while using
+Quilt, please capture the following logs and attach to an email
+to support@quiltdata.io:
+
+1. Browser logs (isolating the most recent request that triggered an error):
+    1. Go to the affected page in your Quilt Catalog.
+    1. Open the browser Developer tools:
+        - Google Chrome: Press **F12**, **Ctrl+Shift+I** or from the
+        Chrome menu select **More tools > Developer tools**.
+        - Mozilla Firefox: Press **F12** or go to **Tools > Browser Tools > Web Developer Tools**.
+        - Apple Safari: Go to **Develop > Show Web Inspector**.
```

Comment:
Thank you for going the extra mile but I'd say docs for multiple browser. It is indeed helpful to say "(e.g. Do X in Chrome)" and use Chrome throughout as an example, but the browser specifics details may change and it's better to let readers figure out what to do on their own. So we mostly need a skeleton with Chrome as the sole example: clear logs, repro issue, save logs. And don't forget that we also want the browser console logs as well, not just browser network logs.